jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/entity/ExternalPolicies.java

```diff
@@ -60,14 +60,19 @@ public static synchronized ExternalPolicies getInstance() {
      * @param dataSource the data source
      */
     public void uploadPolicyDefinition(String dataSource) {
-        LOGGER.info("Started upload policy definition for {}", dataSource);
+        LOGGER.info("Started upload external policies definition for {}", dataSource);
```

Comment:
same here, CE doesn't yet know about external policy concept, this naming should be generic.

Reply:
got it, fixed